src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/ISecurityElement.cs

```diff
@@ -0,0 +1,15 @@
+
+
```

Comment:
Can you just check all the files for extra blank lines at the top so I don't need to point out each one?